README.md

```diff
@@ -10,126 +10,121 @@
 
 This SDK is still considered work in progress.
 
-**With v1.1.0:**
+**For v1 of the SDK, see** [CloudEvents Go SDK v1](./README_v1.md).
 
-[Master](https://github.com/cloudevents/sdk-go/tree/master) will now be the base
-of the effort for v2.0.0 of this SDK and will contain breaking changes or
-missing libraries.
+**v2.0.0-preview2:**
 
-Future work on v1.X.Y releases will branch off of
-[release-1.y.z](https://github.com/cloudevents/sdk-go/tree/release-1.y.z). To
-add a bugfix to a v1.X.Y release, please make a PR to that branch and we can do
-releases as needed on the v1 SDK. No date for EOL on v1 support yet, that will
-be determined by the progress on v2.
+In _preview2_ we are focusing on the new Client interface:
 
-The CloudEvents golang team is working hard to bring you v2.0.0 of the SDK.
-
-**With v1.0.0:**
+```go
+type Client interface {
+	Send(ctx context.Context, event event.Event) error
+	Request(ctx context.Context, event event.Event) (*event.Event, error)
+	StartReceiver(ctx context.Context, fn interface{}) error
+}
+```
 
-The API that exists under [`pkg/cloudevents`](./pkg/cloudevents) will follow
-semver rules. This applies to the root [`./alias.go`](./alias.go) file as well.
+Where a full `fn` looks like
+`func(context.Context, event.Event) (*event.Event, transport.Result)`
 
-Even though `pkg/cloudevents` is v1.0.0, there could still be minor bugs and
-performance issues. We will continue to track and fix these issues as they come
-up. Please file a pull request or issue if you experience problems.
+For protocols that do not support responses, `StartReceiver` will throw an error
+when attempting to set a receiver fn with that capability.
 
-The API that exists under [`pkg/bindings`](./pkg/bindings) is a new API that
-will become SDK v2.x, and will replace `pkg/cloudevents`. This area is still
-under heavy development and will not be following the same semver rules as
-`pkg/cloudevents`. If a release is required to ship changes to `pkg/bindings`, a
-bug fix release will be issued (x.y.z+1).
+For protocols that do not support responses from send (Requester interface),
+`Client.Request` will throw an error.
 
-We will target ~2 months of development to release v2 of this SDK with an end
-date of March 27, 2020. You can read more about the plan for SDK v2 in the
-[SDK v2 planning doc](./docs/SDK_v2.md).
+**v2.0.0-preview1:**
```

Comment:
Don't forget to update it to preview2 :wink:

Reply:
this was for preview1 that we cut last week. preview2 is for this week.